internal/generate/clusterserviceversion/clusterserviceversion.go

```diff
@@ -169,16 +167,7 @@ func (g *Generator) generate() (base *operatorsv1alpha1.ClusterServiceVersion, e
 	if g.FromVersion != "" {
 		base.Spec.Replaces = genutil.MakeCSVName(g.OperatorName, g.FromVersion)
 	}
-	if len(g.RelatedImages) > 0 {
-		base.Spec.RelatedImages = make([]operatorsv1alpha1.RelatedImage, 0, len(g.RelatedImages))
-		for name, image := range g.RelatedImages {
-			base.Spec.RelatedImages = append(base.Spec.RelatedImages, operatorsv1alpha1.RelatedImage{Name: name, Image: image})
-		}
-		// ensure deterministic order
-		sort.SliceStable(base.Spec.RelatedImages, func(i, j int) bool {
-			return strings.Compare(base.Spec.RelatedImages[i].Name, base.Spec.RelatedImages[j].Name) > 0
-		})
-	}
+	base.Spec.RelatedImages = g.RelatedImages
```

Comment:
We might still want to sort to ensure deterministic order if the original deployments/containers/envs are re-ordered. Or is that sort moved somewhere else?